db/statistics.go

```diff
@@ -1526,10 +1634,14 @@ func WriteChartSeriesForDay(day int64) error {
 		}
 	}
 
-	logger.Infof("Exporting MARKET_CAP: %v", newEmission.Div(decimal.NewFromInt(1e18)).Add(decimal.NewFromFloat(72009990.50)).Mul(decimal.NewFromFloat(price.GetEthPrice("USD"))).String())
-	err = SaveChartSeriesPoint(dateTrunc, "MARKET_CAP", newEmission.Div(decimal.NewFromInt(1e18)).Add(decimal.NewFromFloat(72009990.50)).Mul(decimal.NewFromFloat(price.GetEthPrice("USD"))).String())
-	if err != nil {
-		return fmt.Errorf("error calculating MARKET_CAP chart_series: %w", err)
+	switch utils.Config.Chain.Config.DepositChainID {
+	case 1:
+		crowdSale := 72009990.50
+		logger.Infof("Exporting MARKET_CAP: %v", newEmission.Div(decimal.NewFromInt(1e18)).Add(decimal.NewFromFloat(crowdSale)).Mul(decimal.NewFromFloat(price.GetEthPrice("USD"))).String())
```

Comment:
As used twice, we should use a variable for `newEmission.Div(decimal.NewFromInt(1e18)).Add(decimal.NewFromFloat(crowdSale)).Mul(decimal.NewFromFloat(price.GetEthPrice("USD"))).String()`

Reply:
its the sane for all variables throughout the whole function

eg: 
- https://github.com/gobitfly/eth2-beaconchain-explorer/blob/21451456af9d9661be2f06100324764047b7b7dc/db/statistics.go#L1630
- https://github.com/gobitfly/eth2-beaconchain-explorer/blob/21451456af9d9661be2f06100324764047b7b7dc/db/statistics.go#L1622
- etc

i think we should be consistent here